tensorflow/contrib/cudnn_rnn/python/ops/cudnn_rnn_ops.py

```diff
@@ -970,6 +1048,7 @@ def _cudnn_rnn(inputs,
                direction=CUDNN_RNN_UNIDIRECTION,
                dropout=0.,
                seed=0,
+               num_proj=None,
```

Comment:
Please move the new parameter the end for backward compatibility.

Reply:
Even after the param `name`?